style.css

```diff
@@ -0,0 +1,147 @@
+@import url(https://chess-ui.netlify.app/components/components.css);
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    font-size: 62.5%;
+    --COLOR-SHADOW-4: #1b1b1b9d;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+/* LANDING PAGE */
+
+.nav {
+    display: flex;
+}
+
+.box-shadow-100 {
+    box-shadow: 0 5px 2.5rem 2px var(--PRIMARY-50);
+}
+
+/* DOCUMENTATION */
+
+.grand-body {
+    position: relative;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto 1fr;
+    grid-template-areas:
+        " navbar navbar"
+        " main main";
+    background-color: var(--COLOR-SHADOW-PRIMARY-50);
+    min-height: 100vh;
+    max-height: 100vh;
+}
+
+.grand-nav {
+    position: sticky;
+    top: 0;
+    grid-area: navbar;
+    z-index: 2;
```

Comment:
better to use variables for z-index

Reply:
Thanks for suggestions Sachin. I used direct values because I have used z-index at only 2 or 3 places throughout the project so won't be an issue as such. But if I add more z-index values will surely create variables for them.